src/dependency-manager/spec/service-spec.ts

```diff
@@ -257,4 +257,11 @@ export class ServiceSpec extends ResourceSpec {
   @ValidateNested()
   @Type(() => DeploySpec)
   deploy?: DeploySpec;
+
+  @IsOptional()
+  @JSONSchema({
+    ...ExpressionOrString(),
+    description: `A period of time between a service being passed a SIGINT and a SIGTERM when it's scheduled to be replaced or terminated. Only used for remote deployments.`,
```

Comment:
Can you also add the default to the JSONSchema?
ex. https://github.com/architect-team/architect-cli/blob/58225bfff274b98b711058b7ffc2f55d01e04ad3/src/dependency-manager/src/spec/common-spec.ts#L67

Reply:
That's there. I forgot at first but must have gotten to it before this comment landed